README.md

```diff
@@ -0,0 +1,41 @@
+# Northcoders News
+
+A social news aggregation, content-rating and discussion website.
+
+Visit https://nc-news-five.vercel.app to see a live demo.
+
+## Background
+This project formed the basis of a second week-long sprint on the Northcoders software development bootcamp.
+
+The intention of the sprint was to mimic the building of a real world front end project to consume the API developed in the back end project.
+
```

Comment:
Mayhaps you could share a link to your API

Reply:
Great idea, thank you!!